### ISSUE TITLE
⌨ Dashboards all have same title

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#11978]
## ⌨ Accessibility Issue

The dashboards all have the same title "Welcome back, {name}". When switching roles, this makes things very confusing for a screen reader user.

## 🦋 Expected Behaviour

It is clear to users that when switching roles, they have navigated to a new, unique, page.

## 📋 Steps to Reproduce

1. Login as a manager `manager@test.com`
2. Navigate to dashboard link
3. Observe the title being read out
4. Navigate to the role switcher and switch to the applicant role
5. Observe the same title from step 3 being read out

## 🙋‍♀️ Proposed Solution

Possibly use a different title to distinguish the different types of dashboards. We could use the welcome message as a subtitle 🤔 

## ✅ Acceptance Criteria

- [ ] Each dashboard (Applicant, Community and Admin) should have a different HTML page title: [Role] dashboard - GC Digital Talent
- [ ] Add hidden text to the h1 elements of the dashboards: 
  - Welcome back < hidden> to your community dashboard </ hidden>, username
  - Welcome back < hidden> to your admin dashboard, username
  - Welcome back < hidden> to your applicant dashboard</ hidden>, username

**Comment by tristan-orourke** [comment]
@JoshBeveridge could we get your consideration?

**Comment by JoshBeveridge** [comment]
The page title and the hero title should be different in this case, no? I was under the assumption we had "[Role] dashboard - GC Digital Talent" as the HTML page title. Is this not being read out on page load?

**Comment by esizer** [comment]
> The page title and the hero title should be different in this case, no? I was under the assumption we had "[Role] dashboard - GC Digital Talent" as the HTML page title. Is this not being read out on page load?

No, since we have an SPA page title is not read out on page load since a page change never occurs once you load the app. We simply force focus to the `h1` of the page and read that.

**Comment by tristan-orourke** [comment]
@NienkeBr help us decide on unique h1 titles for the Community and Admin dashboards

**Comment by esizer** [comment]
> Would it be a somewhat acceptable solution to include hidden text for screen readers? Or would you advise against that @esizer? For example:
> 
> * Welcome back < hidden> to your community dashboard </ hidden>, username
> * Welcome back < hidden> to your admin dashboard, username
> * Welcome back < hidden> to your applicant dashboard</ hidden>, username

It could be, the other issue is from a visual perspective, we use the exact same page so when you change from one dashboard to another it even looks like you are on the same page and seems like the link did nothing :sweat_smile:

**Comment by NienkeBr** [comment]
Yes, at least for now between community and admin it's extremely similar. However, that's supposed to change at some point with the new community dashboard. Would a unique subtitle/description address the visual perspective? For the community dashboard I'd like to change that already now anyway as it says that you're on the admin hub. I'll create a separate ticket for that.

**Comment by tristan-orourke** [comment]
Resolved by #12209

**Comment by esizer** [comment]
> Resolved by #12209

This was not resolved with that ticket. The titles remain the same and screen reader users are still unaware a navigation happened at all.
